libs/RobotKit/include/RobotController.h

```diff
@@ -25,11 +25,34 @@ class RobotController : public interface::RobotController
 	explicit RobotController(interface::Timeout &sleep_timeout, interface::Battery &battery)
 		: _sleep_timeout(sleep_timeout), _battery(battery) {};
 
+	void splashScreen() final
```

Comment:
```suggestion
	void displaySplashScreen() final
```

pour info, sous iOS ils appellent ça launch screen.